File: src/modules/task/entity/task.entity.ts
import { Column, Entity } from 'typeorm';
import { BaseEntity } from '../../../common/entity/base.entity';

@Entity({ name: 'task' })
export class TaskEntity extends BaseEntity {
  @Column({
    name: 'title',
    type: 'varchar',
  })
  title: string;

  @Column({
    name: 'content',
    type: 'varchar',
  })
  content: string;

  @Column({
    name: 'user_id',
    type: 'uuid',
  })
  userId: string;
}

File: src/main/java/com/manabie/todo/entity/TaskEntity.java
package com.manabie.todo.entity;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.hibernate.annotations.CreationTimestamp;
import org.hibernate.annotations.UpdateTimestamp;

import javax.persistence.*;
import java.sql.Timestamp;

@Data
@Entity
@Table(name = "task")
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class TaskEntity {
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  private Long id;

  private Long userId;

  private String title;

  private String description;

  @CreationTimestamp
  private Timestamp datetimeCreated;

  @UpdateTimestamp
  private Timestamp datetimeEdited;

  @Builder.Default
  private Boolean isDelete = false;
}
